libs/datamodel/core/src/error/collection.rs

```diff
@@ -1,4 +1,4 @@
-use super::DatamodelError;
+use crate::error::DatamodelError;
```

Comment:
I think the `super` way of importing this is a bit clearer. `super` indicates that the type can be found within the same module/folder. With the other import the reader has to first understand that the absolute import refers to the module we already in.